README.md

```diff
@@ -1,4 +1,16 @@
-[![blackbird-logo](https://raw.githubusercontent.com/blackbird-cloud/terraform-module-template/main/.config/logo_simple.png)](https://blackbird.cloud)
+<!-- BEGIN_TF_DOCS -->
+# AWS Account information Terraform module
+A Terraform module which uses a number of datasources to read information about the current AWS account.
```

Comment:
Include reason why this is useful, maybe something like "It's useful on a terragrunt scenario where the output of this module can be used for other things rather than hard coding values"